src/containers/CountryView/index.tsx

```diff
@@ -283,12 +287,18 @@ const CountryView: React.FC = () => {
             'admin-1-boundary',
         );
 
+        //Filter out countries without any data
+        mapRef.setFilter('countries-join', [
+            'in',
+            'iso_3166_1_alpha_3',
+            ...countriesData.map((country) => country['name']),
```

Comment:
Here you can just use `country.name`. After using dot Typescript will show you possible fields that you can use in this object. Using array style expression won't give you autocomplete benefits